services/online_exam_service/.env

```
@@ -1,18 +1,28 @@
-# DATABASE CONFIG (k?t n?i d?n course_db)
+# PostgreSQL Database (shared with exam-service)
 DB_HOST=localhost
 DB_PORT=5433
 DB_USER=postgres
 DB_PASSWORD=password
 DB_NAME=exam_db
 
-# BLOCKCHAIN CONFIG
+# Blockchain (Ganache)
 WEB3_PROVIDER_URL=http://127.0.0.1:7545
+OWNER_ACCOUNT_PRIVATE_KEY=0xec22a90f77ee5551ab275d723ef70c72f3adf51cba2384a55b4d6d12657e6fa6
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The OWNER_ACCOUNT_PRIVATE_KEY key should go before the WEB3_PROVIDER_URL key

(UnorderedKey)